DigitalLearningSolutions.Web/Views/LearningContent/_BrandCourseCard.cshtml

```diff
@@ -22,29 +31,24 @@
           <span hidden name="popularity-score">@Model.PopularityRating</span>
         </div>
       </div>
-      <span hidden data-filter-value="@Model.CategoryFilter">
-        @Model.CategoryName
-      </span>
-      <span hidden data-filter-value="@Model.TopicFilter">
-        @Model.CourseTopic
-      </span>
+      
       <span class="nhsuk-u-margin-bottom-4">
         <span class="nhsuk-u-font-weight-bold">Created</span> @Model.CreatedDate.ToString(DateHelper.StandardDateFormat)
         <span hidden data-name-for-sorting="created-date">@Model.CreatedDate</span>
       </span>
 
-      <div>
-        <div class="header-row nhsuk-u-font-weight-bold">
-          <div class="value header-row-content">Section</div>
-          <div class="value header-row-content">Tutorials</div>
-        </div>
+      <table>
+        <tr class="header-row nhsuk-u-font-weight-bold">
+          <th class="value header-row-content">Section</th>

```

Comment:
The NHS tables have specific roles for different table elements. Do we want them here?
https://service-manual.nhs.uk/design-system/components/table

Reply:
I don't believe we need them since we are using the correct html elements, looking at the various bits of documentation on them e.g. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/row_role.
e.g. I probably should have had them for the table structure with divs that I had before.